oe_corporate_blocks.post_update.php

```diff
@@ -377,3 +377,54 @@ function oe_corporate_blocks_post_update_50001(): void {
   $config_path = \Drupal::service('extension.list.module')->getPath('oe_corporate_blocks') . '/config/post_update/50001_update_footer_data';
   _oe_corporate_blocks_import_corporate_links($config_path);
 }
+
+/**
+ * Update EC footer data.
+ */
+function oe_corporate_blocks_post_update_50002(): void {
+  $config_path = \Drupal::service('extension.list.module')->getPath('oe_corporate_blocks') . '/config/post_update/50002_update_footer_data';
+  $config_name = 'oe_corporate_blocks.ec_data.footer';
+
+  // Delete the config translations.
```

Comment:
There is one string that is duplicated, so I'm not sure I understand why are all the config translations being deleted?